[BUG] After Upgrade to 1.10.0 - libatomic.so.1: cannot open shared object file: No such file or directory

### Description

After the upgrade to Docker Version 1.10.0, when consuming files an error occurs. Saying the libatomic is not found: 

20221128-070606.pdf: The following error occurred while consuming 20221128-070606.pdf: libatomic.so.1: cannot open shared object file: No such file or directory

### Steps to reproduce

Add new document using import directory

### Webserver logs

```bash
[2022-11-28 08:12:27,159] [ERROR] [paperless.consumer] The following error occurred while consuming 28112022-070649.pdf: libatomic.so.1: cannot open shared object file: No such file or directory

Traceback (most recent call last):

  File "/usr/src/paperless/src/documents/consumer.py", line 383, in try_consume_file

    document_consumption_finished.send(

  File "/usr/local/lib/python3.9/site-packages/django/dispatch/dispatcher.py", line 176, in send

    return [

  File "/usr/local/lib/python3.9/site-packages/django/dispatch/dispatcher.py", line 177, in <listcomp>

    (receiver, receiver(signal=self, sender=sender, **named))

  File "/usr/src/paperless/src/documents/signals/handlers.py", line 56, in set_correspondent

    potential_correspondents = matching.match_correspondents(document, classifier)

  File "/usr/src/paperless/src/documents/matching.py", line 30, in match_correspondents

    return list(

  File "/usr/src/paperless/src/documents/matching.py", line 31, in <lambda>

    filter(lambda o: matches(o, document) or o.pk == pred_id, correspondents),

  File "/usr/src/paperless/src/documents/matching.py", line 145, in matches

    from rapidfuzz import fuzz

  File "/usr/local/lib/python3.9/site-packages/rapidfuzz/__init__.py", line 8, in <module>

    from rapidfuzz import distance, fuzz, process, string_metric, utils

  File "/usr/local/lib/python3.9/site-packages/rapidfuzz/process.py", line 9, in <module>

    extract = _fallback_import(_mod, "extract")

  File "/usr/local/lib/python3.9/site-packages/rapidfuzz/_utils.py", line 71, in fallback_import

    cpp_mod = importlib.import_module(module + "_cpp")

  File "/usr/local/lib/python3.9/importlib/__init__.py", line 127, in import_module

    return _bootstrap._gcd_import(name[level:], package, level)

  File "/usr/local/lib/python3.9/site-packages/rapidfuzz/process_cpp.py", line 9, in <module>

    from rapidfuzz.process_cpp_impl import FLOAT32 as _FLOAT32

ImportError: libatomic.so.1: cannot open shared object file: No such file or directory

[2022-11-28 08:12:27,167] [DEBUG] [paperless.parsing.tesseract] Deleting directory /tmp/paperless/paperless-qd6dz7_6
```


### Browser logs

_No response_

### Paperless-ngx version

10.0

### Host OS

Raspberry Pi 64

### Installation method

Docker - official image

### Browser

Firefox

### Configuration changes

_No response_

### Other

_No response_

can confirm the bug. i also geht it when i use the in the correspondent page the show documents button and correspondent suggestions is also not working.

```
[2022-11-28 23:06:49,012] [ERROR] [django.request] Internal Server Error: /api/documents/3385/suggestions/
Traceback (most recent call last):
  File "/usr/local/lib/python3.9/site-packages/asgiref/sync.py", line 472, in thread_handler
    raise exc_info[1]
  File "/usr/local/lib/python3.9/site-packages/django/core/handlers/exception.py", line 42, in inner
    response = await get_response(request)
  File "/usr/local/lib/python3.9/site-packages/django/core/handlers/base.py", line 253, in _get_response_async
    response = await wrapped_callback(
  File "/usr/local/lib/python3.9/site-packages/asgiref/sync.py", line 435, in __call__
    ret = await asyncio.wait_for(future, timeout=None)
  File "/usr/local/lib/python3.9/asyncio/tasks.py", line 442, in wait_for
    return await fut
  File "/usr/local/lib/python3.9/site-packages/asgiref/current_thread_executor.py", line 22, in run
    result = self.fn(*self.args, **self.kwargs)
  File "/usr/local/lib/python3.9/site-packages/asgiref/sync.py", line 476, in thread_handler
    return func(*args, **kwargs)
  File "/usr/local/lib/python3.9/site-packages/django/views/decorators/csrf.py", line 54, in wrapped_view
    return view_func(*args, **kwargs)
  File "/usr/local/lib/python3.9/site-packages/rest_framework/viewsets.py", line 125, in view
    return self.dispatch(request, *args, **kwargs)
  File "/usr/local/lib/python3.9/site-packages/rest_framework/views.py", line 509, in dispatch
    response = self.handle_exception(exc)
  File "/usr/local/lib/python3.9/site-packages/rest_framework/views.py", line 469, in handle_exception
    self.raise_uncaught_exception(exc)
  File "/usr/local/lib/python3.9/site-packages/rest_framework/views.py", line 480, in raise_uncaught_exception
    raise exc
  File "/usr/local/lib/python3.9/site-packages/rest_framework/views.py", line 506, in dispatch
    response = handler(request, *args, **kwargs)
  File "/usr/src/paperless/src/documents/views.py", line 354, in suggestions
    dt.id for dt in match_document_types(doc, classifier)
  File "/usr/src/paperless/src/documents/matching.py", line 43, in match_document_types
    return list(
  File "/usr/src/paperless/src/documents/matching.py", line 44, in <lambda>
    filter(lambda o: matches(o, document) or o.pk == pred_id, document_types),
  File "/usr/src/paperless/src/documents/matching.py", line 145, in matches
    from rapidfuzz import fuzz
  File "/usr/local/lib/python3.9/site-packages/rapidfuzz/__init__.py", line 8, in <module>
    from rapidfuzz import distance, fuzz, process, string_metric, utils
  File "/usr/local/lib/python3.9/site-packages/rapidfuzz/process.py", line 9, in <module>
    extract = _fallback_import(_mod, "extract")
  File "/usr/local/lib/python3.9/site-packages/rapidfuzz/_utils.py", line 71, in fallback_import
    cpp_mod = importlib.import_module(module + "_cpp")
  File "/usr/local/lib/python3.9/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "/usr/local/lib/python3.9/site-packages/rapidfuzz/process_cpp.py", line 9, in <module>
    from rapidfuzz.process_cpp_impl import FLOAT32 as _FLOAT32
ImportError: libatomic.so.1: cannot open shared object file: No such file or directory

```

using it with docker on a rpi 4 with armv7 32 bit


```apt-get update && apt-get install libatomic1``` and container restart solves the problem

--- a/Dockerfile
+++ b/Dockerfile
@@ -127,7 +127,9 @@ ARG RUNTIME_PACKAGES="\
   zlib1g \
   # Barcode splitter
   libzbar0 \
-  poppler-utils"
+  poppler-utils \
+  # RapidFuzz on armv7
+  libatomic1"
 
 # Install basic runtime packages.
 # These change very infrequently

--- a/Dockerfile
+++ b/Dockerfile
@@ -72,34 +72,37 @@ COPY --from=jbig2enc-builder /usr/src/jbig2enc/src/*.h /usr/local/include/
 
 # Packages need for running
 ARG RUNTIME_PACKAGES="\
+  # Python
+  python3 \
+  python3-pip \
+  python3-setuptools \
+  # General utils
   curl \
-  file \
+  # Docker specific
+  gosu \
+  # Timezones support
+  tzdata \
   # fonts for text file thumbnail generation
   fonts-liberation \
   gettext \
   ghostscript \
   gnupg \
-  gosu \
   icc-profiles-free \
   imagemagick \
-  media-types \
+  # Image processing
   liblept5 \
-  libpq5 \
-  libxml2 \
   liblcms2-2 \
   libtiff5 \
-  libxslt1.1 \
   libfreetype6 \
   libwebp6 \
   libopenjp2-7 \
   libimagequant0 \
   libraqm0 \
-  libgnutls30 \
   libjpeg62-turbo \
-  python3 \
-  python3-pip \
-  python3-setuptools \
+  # PostgreSQL
+  libpq5 \
   postgresql-client \
+  # MySQL / MariaDB
   mariadb-client \
   # For Numpy
   libatlas3-base \
@@ -110,13 +113,17 @@ ARG RUNTIME_PACKAGES="\
   tesseract-ocr-fra \
   tesseract-ocr-ita \
   tesseract-ocr-spa \
-  # Suggested for OCRmyPDF
-  pngquant \
-  # Suggested for pikepdf
-  jbig2dec \
-  tzdata \
   unpaper \
+  pngquant \
+  # pikepdf / qpdf
+  jbig2dec \
+  libxml2 \
+  libxslt1.1 \
+  libgnutls30 \
   # Mime type detection
+  file \
+  libmagic1 \
+  media-types \
   zlib1g \
   # Barcode splitter
   libzbar0 \

